test/e2e_node/util.go

```diff
@@ -87,7 +88,11 @@ func getNodeSummary(ctx context.Context) (*stats.Summary, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get current kubelet config")
 	}
-	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/stats/summary", net.JoinHostPort(kubeletConfig.Address, strconv.Itoa(int(kubeletConfig.ReadOnlyPort)))), nil)
+	readonlyPort := int(kubeletConfig.ReadOnlyPort)
+	if readonlyPort == 0 {
+		e2eskipper.Skipf("Skipping /stats/summary check as the read-only port for the Kubelet is disabled")
+	}
```

Comment:
can we not provide a credential so the non-readonly port can be used? how many e2e tests is this skipping?

Reply:
I remove the getNodeSummary using 10255 here and change it to use `e2edebug.GetStatsSummary`.